vue-components/tests/unit/components.spec.ts

```diff
@@ -3,8 +3,17 @@ import * as allComponents from '@/main';
 
 describe( 'All components', () => {
 	it( 'have the class wikit in their root node', () => {
+		// props to feed to certain components so they can be built
+		// TODO in order not to maintain this forever, consider switching this to an eslint-based solution
```

Comment:
I'm not a big fan of inline TODO as they tend to get forgotten and fall out of sync with their surrounding code. Can you please open up a ticket in the DS Phab board to consider the eslint method, that way it will also be prioritized.

Reply:
Created the task: https://phabricator.wikimedia.org/T267741
And explicitly mentioned it in the comment, so readers of the comment can directly evaluate whether it is still relevant or can be removed.